demos/twa-play-billing/src/main/java/com/google/androidbrowserhelper/demos/playbilling/BillingDelegationService.java

```diff
@@ -0,0 +1,41 @@
+package com.google.androidbrowserhelper.demos.playbilling;
+
+import android.os.Bundle;
+
+import com.google.androidbrowserhelper.trusted.DelegationService;
+
+import androidx.annotation.NonNull;
+import androidx.annotation.Nullable;
+import androidx.browser.trusted.TrustedWebActivityCallbackRemote;
+
+public class BillingDelegationService extends DelegationService {
+    public static final String KEY_SUCCESS = "success";
+
+    @Nullable
+    @Override
+    public Bundle onExtraCommand(@NonNull String commandName, @NonNull Bundle args,
+                                 @Nullable TrustedWebActivityCallbackRemote callbackRemote) {
+        Bundle bundle = new Bundle();
+        bundle.putBoolean(KEY_SUCCESS, false);
+
+        switch (commandName) {
```

Comment:
If I understood correctly so far, this bit of the code will receive the messages from the browser (get detail and acknowledge) and dispatch to the appropriate call implementation. The call implementation will handle the conversation with the billing API. 

I feel `GetDatailsCall` and `AcknowledgeCall` are similar and should implement a common interface. May be wrong though.